code/modules/clothing/suits/armor.dm

```diff
@@ -20,21 +20,21 @@
 	heat_protection = UPPER_TORSO|LOWER_TORSO
 	armor = list(melee = 45, bullet = 40, laser = 50, energy = 20, bomb = 25, bio = 0, rad = 0)
 
-/obj/item/clothing/suit/armor/vest/ds_armor
-	name = "security officer's armor"
-	desc = "An armored vest that protects against some damage."
-	icon_state = "ds_securityarmor"
-	//item_state = "armor"
-	body_parts_covered = UPPER_TORSO|LOWER_TORSO
-	cold_protection = UPPER_TORSO|LOWER_TORSO
-	heat_protection = UPPER_TORSO|LOWER_TORSO
-	armor = list(melee = 45, bullet = 40, laser = 50, energy = 0, bomb = 25, bio = 0, rad = 0)
+/obj/item/clothing/suit/armor/vest/pcsi
+	name = "P.C.S.I officer's vest"
+	desc = "The Concordance Extraction Corporation's standard issue security armor, used on its larger vessels and colonial interests, outfitted with protective polymer plating and a lightweight kevlar web beneath those plates. Deployed in high-risk situations where CEC property and personnel are at risk, usually from piracy or looters."
+	icon_state = "pcsi_armor"
```

Comment:
this sprite, with an underlay, is what we'll be using for the PCSI rig. The armor vest will be something else